src/script.js

```diff
@@ -0,0 +1,176 @@
+import './style.css';
+import { clearCompletedItems, handleCheckboxClick } from './modules/interactive.js';
+
+class TaskManager {
+  constructor() {
+    this.myTasks = [];
+    this.container = document.getElementById('list-items');
+    this.myTaskInput = document.getElementById('addtask');
+    this.clearButton = document.querySelector('.clear');
```

Comment:
Please ensure consistent naming conventions throughout the code. For example, some variables use camel case (myTaskInput, clearButton), while others use kebab case (list-items). Kindly do this to improve code clarity.